### ISSUE TITLE
bug: poetry install warning

### ISSUE THREAD

**Issue by introkun** [TreyWW/MyFinances#237]
### What version of MyFinances are you currently using?

v0.2.1

### What device type are you currently facing the issue on?

Desktop

### Describe the bug

```
Installing the current project: MyFinances (0.2.1)
Warning: The current project could not be installed: No file/folder found for package myfinances
If you do not want to install the current project use --no-root.
If you want to use Poetry only for dependency management but not for packaging, you can disable package mode by setting package-mode = false in your pyproject.toml file.
In a future version of Poetry this warning will become an error!
```

**Comment by TreyWW** [comment]
Yeah I get this error often too, I just ignore it. It still works fine, just a warning. I'm not too sure how to "hide" it, any suggestions are welcome :)

**Comment by introkun** [comment]
We should use `poetry install --no-root`, not poetry install. Since the `install` command will install dependencies and the project, but it's not needed.
@TreyWW could you please update the documentation?

**Comment by TreyWW** [comment]
Sure thing, i'll put it on my list (feel free to create a PR though - if not I'll create it tomorrow).

Thanks

**Comment by introkun** [comment]
Ok, I'll update whenever I can

**Comment by TreyWW** [comment]
Fixed in #256
